artman/tasks/gapic_tasks.py

```diff
@@ -34,11 +34,13 @@ def execute(self, toolkit_path, descriptor_set, service_yaml,
         self.exec_command(['mkdir', '-p', config_gen_dir])
         config_gen_path = os.path.join(config_gen_dir,
                                        api_full_name + '_gapic.yaml')
-        service_args = ['--service_yaml=' + os.path.abspath(service_yaml)]
         args = [
             '--descriptor_set=' + os.path.abspath(descriptor_set),
             '--output=' + os.path.abspath(config_gen_path)
-        ] + service_args
+        ]
+        if service_yaml != '':
```

Comment:
If I understand correctly, empty strings in Python are falsy, so you could just write:

`if service_yaml`: